### PR TITLE
[ES] Change date and references format

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`SKIP_OUTDATED_CHECK`

### CODE REVIEW

File: wiki/History_of_osu!/2016/es.md [hunk 1]
```diff
@@ -151,32 +151,32 @@ A pesar de las primeras adiciones, se recordó a los usuarios que el nuevo estad
...
-[^osu-weekly-57]: [Publicación de noticia por Nyquill (20-04-2016) "osu!weekly #57"](https://osu.ppy.sh/home/news/2016-04-20-osuweekly-57)
+[^mapset-restructure-proposal]: [Hilo del foro por ztrot (13/2/2016) «Mapset and Spread Restructure (Proposed)»](https://osu.ppy.sh/community/forums/topics/420223)
+[^meet-ephemeral]: [Publicación de blog por ppy (17/2/2016) «Meet Ephemeral!»](https://blog.ppy.sh/post/139478794378/meet-ephemeral)
+[^mv2-timeline-doc]: [Documento de Google por Ephemeral (17/2/2016) «Timeline of Events Surrounding Recent Changes»](https://docs.google.com/document/d/1VlFUIte8ho4tssRCucSBt96nTVAhsCLfJZcFVgzHjuk/edit)
+[^osu-weekly-48]: [Publicación de noticia por Nyquill (16/2/2016) «osu!weekly #48»](https://osu.ppy.sh/home/news/2016-02-16-osuweekly-48)
+[^osu-go-post]: [Hilo del foro por DW10 (26/2/2016) «osu!go not worth it?»](https://osu.ppy.sh/community/forums/topics/425108)
+[^meeting-notes/2016-feb]: [Publicación de blog por ppy (9/3/2016) «2016-02 meeting notes»](https://blog.ppy.sh/post/140259300353/2016-02-meeting-notes)
```
Comment (agatemosu) [inline wiki/History_of_osu!/2016/es.md:161]:
```suggestion
[^meeting-notes-2016-feb]: [Publicación de blog por ppy (9/3/2016) «2016-02 meeting notes»](https://blog.ppy.sh/post/140259300353/2016-02-meeting-notes)
```

File: wiki/History_of_osu!/2016/es.md [hunk 2]
```diff
@@ -151,32 +151,32 @@ A pesar de las primeras adiciones, se recordó a los usuarios que el nuevo estad
...
+[^meet-ephemeral]: [Publicación de blog por ppy (17/2/2016) «Meet Ephemeral!»](https://blog.ppy.sh/post/139478794378/meet-ephemeral)
+[^mv2-timeline-doc]: [Documento de Google por Ephemeral (17/2/2016) «Timeline of Events Surrounding Recent Changes»](https://docs.google.com/document/d/1VlFUIte8ho4tssRCucSBt96nTVAhsCLfJZcFVgzHjuk/edit)
+[^osu-weekly-48]: [Publicación de noticia por Nyquill (16/2/2016) «osu!weekly #48»](https://osu.ppy.sh/home/news/2016-02-16-osuweekly-48)
+[^osu-go-post]: [Hilo del foro por DW10 (26/2/2016) «osu!go not worth it?»](https://osu.ppy.sh/community/forums/topics/425108)
+[^meeting-notes-2016-feb]: [Publicación de blog por ppy (9/3/2016) «2016-02 meeting notes»](https://blog.ppy.sh/post/140259300353/2016-02-meeting-notes)
+[^mv2-feedback-thread]: [Hilo del foro por Loctab (11/4/2016) «Modding v2: Modding Panel - Feedback Thread»](https://osu.ppy.sh/community/forums/topics/442285)
+[^osu-weekly-57]: [Publicación de noticia por Nyquill (20/4/2016) «osu!weekly #57»](https://osu.ppy.sh/home/news/2016-04/20-osuweekly-57)
```
Comment (agatemosu) [inline wiki/History_of_osu!/2016/es.md:163]:
```suggestion
[^osu-weekly-57]: [Publicación de noticia por Nyquill (20/4/2016) «osu!weekly #57»](https://osu.ppy.sh/home/news/2016-04-20-osuweekly-57)
```

**Review by TicClick — APPROVED** [review]
don't know Spanish but the changes look alright. is there any specific reason behind changing the format?

File: wiki/History_of_osu!/2011/es.md [hunk 3]
```diff
@@ -10,4 +6,4 @@ Se anunció la osu! wiki, que permite a los usuarios agregar todo tipo de inform
 
 Links:
```
Comment (agatemosu) [inline wiki/History_of_osu!/2011/es.md:7]:
```suggestion
Enlaces:
```

**Comment by agatemosu** [comment]
> don't know Spanish but the changes look alright. is there any specific reason behind changing the format?

Because the page of the Spanish academy recommends [this](https://www.rae.es/dpd/fecha#:~:text=Cuando%20el%20n%C3%BAmero%20que%20indica%20el%20mes%20o%20el%20d%C3%ADa%20es%20inferior%20a%20diez%2C%20se%20recomienda%20no%20anteponer%20un%20cero%20a%20la%20cifra%20simple%2C%20salvo%20que%20ello%20sea%20necesario%20por%20razones%20t%C3%A9cnicas): "When the number indicating the month or day is less than ten, it is recommended not to place a zero before the single digit, unless this is necessary for technical reasons."

**Review by TicClick — APPROVED** [review]

**Pull request merged**